CHANGELOG.md

```diff
@@ -0,0 +1,1340 @@
+# Changelog
+
+## [3.1.2](https://github.com/AntiMicroX/antimicrox/tree/3.1.2) (2020-09-30)
+
+[Full Changelog](https://github.com/AntiMicroX/antimicrox/compare/3.1.1...3.1.2)
+
+**Fixed bugs:**
+
+- Right click - Turbo and Clear not working properly [\#7](https://github.com/AntiMicroX/antimicrox/issues/7)
+- Wrong order for setting sets [\#6](https://github.com/AntiMicroX/antimicrox/issues/6)
+
+**Closed issues:**
```

Comment:
I think Closed issues shouldn't be listed here

Reply:
But I think they would look good in Release description.
I will investigate merging it with CI.

Reply:
Ok, I'll remove that from the config.